### PR TITLE
feat: configure agencies list in the manifest

### PR DESCRIPTION
**Pull request by doubleface**
Exports a manifestConfig.json file which can be copied in the src/conf/konnectors.json file in the collect application

### CODE REVIEW

File: src/regions.js [hunk 1]
```diff
@@ -23,15 +23,33 @@ request('https://www.credit-agricole.fr').then(body => {
...
   )
+
+  manifestConfig.sort((a, b) => a.name.localeCompare(b.name))
+  fs.writeFileSync(
+    path.join(__dirname, '..', 'manifestConfig.json'),
+    JSON.stringify(manifestConfig, null, 2)
+  )
+
+  console.log(
+    `The manifest configuration is written in ${path.resolve(
```
Comment (ptbrowne) [inline src/regions.js]: Is there a particular reason why you use `path.resolve` here and `path.join` above ? I would use the same method and have the path in a var :) Happy holidays :D 🌲🎅

Comment (doubleface) [reply src/regions.js]: No good reason, my mistake.

**Comment by gregorylegarec** [comment]
Does it impacts our needs for the next konnectors manifests ? https://gist.github.com/gregorylegarec/bc9fa278ee18b63ea7e8ae4bd43961d5

**Comment by doubleface** [comment]
@gregorylegarec No, this generate a combo box like the one already used for some linxo connectors

**Pull request merged**